frontend/src/app/docs/api-docs/api-docs-data.ts

```diff
@@ -9871,7 +9871,292 @@ export const restApiDocsData = [
+          headers: "api_key: stacksats",
+          response: `{
+  "balance": 99900000,
+  "hold": 0,
+  "feesPaid": 200000
+}`,
+        },
+      }
+    }
+  },
+  {
+    options: { officialOnly: true },
+    type: "endpoint",
+    category: "accelerator",
+    httpRequestMethod: "POST",
+    fragment: "estimate",
+    title: "POST Calculate Estimated Costs",
+    description: {
+      default: "<p>Return an estimation of the costs to accelerate a transaction.</p>"
+    },
+    urlString: "/v1/services/accelerator/estimate",
+    showConditions: bitcoinNetworks,
+    showJsExamples: showJsExamplesDefaultFalse,
+    codeExample: {
+      default: {
+        codeTemplate: {
+          curl: `%{1}" "[[hostname]][[baseNetworkUrl]]/api/v1/services/accelerator/estimate`, //custom interpolation technique handled in replaceCurlPlaceholder()
+          commonJS: ``,
+          esModule: ``
+        },
+        codeSampleMainnet: {
+          esModule: [],
+          commonJS: [],
+          curl: ["txInput=ee13ebb99632377c15c94980357f674d285ac413452050031ea6dcd3e9b2dc29"],
+          headers: "api_key: stacksats",
+          response: `{
+  "txSummary": {
+    "txid": "ee13ebb99632377c15c94980357f674d285ac413452050031ea6dcd3e9b2dc29",
+    "effectiveVsize": 154,
+    "effectiveFee": 154,
+    "ancestorCount": 1
+  },
+  "cost": 3850,
+  "targetFeeRate": 26,
+  "nextBlockFee": 4004,
+  "userBalance": 99900000,
+  "mempoolBaseFee": 40000,
+  "vsizeFee": 50000,
+  "hasAccess": true
+}`,
+        },
+      }
+    }
+  },
+  {
+    options: { officialOnly: true },
+    type: "endpoint",
+    category: "accelerator",
+    httpRequestMethod: "POST",
+    fragment: "accelerate",
+    title: "POST Accelerate A Transaction",
+    description: {
+      default: "<p>Send a request to accelerate a transaction.</p>"
+    },
+    urlString: "/v1/services/accelerator/accelerate",
+    showConditions: bitcoinNetworks,
+    showJsExamples: showJsExamplesDefaultFalse,
+    codeExample: {
+      default: {
+        codeTemplate: {
+          curl: `%{1}" "[[hostname]][[baseNetworkUrl]]/api/v1/services/accelerator/accelerate`, //custom interpolation technique handled in replaceCurlPlaceholder()
+          commonJS: ``,
+          esModule: ``
+        },
+        codeSampleMainnet: {
+          esModule: [],
+          commonJS: [],
+          curl: ["txInput=ee13ebb99632377c15c94980357f674d285ac413452050031ea6dcd3e9b2dc29&userBid=21000000"],
+          headers: "api_key: stacksats",
+          response: `HTTP/1.1 200 OK`,
+        },
+      }
+    }
+  },
+  {
+    options: { officialOnly: true },
+    type: "endpoint",
+    category: "history",
+    httpRequestMethod: "GET",
+    fragment: "history",
+    title: "GET Acceleration History",
+    description: {
+      default: "<p>Return the history of previous acceleration requests.</p>"
+    },
+    urlString: "/v1/services/accelerator/history[?status=all|requested|accelerating|mined|completed|failed]",
```

Comment:
I understand what this is trying to say, but it would be more consistent to put the options for the variable in the description ([example](http://localhost:4200/docs/api/rest#get-lightning-network-stats)).